### PR TITLE
rake requires 1.9.3 starting with v11.0.0

### PR DESCRIPTION
**Pull request by kjg**
This should make travis green again on ruby 1.9.2 and below

### CODE REVIEW

**Comment by kjg** [comment]
This has passed Travis now. Please merge so new branches can get accurate Travis runs.

Thanks!

**Comment by kjg** [comment]
@jeremy or @bf4 can you please merge this or something like it to bring master back to green so that future PRs can be properly evaluated for errors.

**Comment by jeremy** [comment]
Yes.

File: gemfiles/mime_types_2.latest.gemfile [hunk 1]
```diff
@@ -1,12 +0,0 @@
...
-source "https://rubygems.org"
-
-gem "jruby-openssl", :platforms => :jruby
-gem "mime-types", "~> 2.3"
-
-group :local_development, :test do
-  gem "appraisal", "~> 1.0"
-end
-
-gemspec :path => "../"
```
Comment (jeremy) [inline gemfiles/mime_types_2.latest.gemfile]: Why'd you remove this gemfile? It guarantees that we're testing the latest 2.x mime-types.

Comment (kjg) [inline gemfiles/mime_types_2.latest.gemfile]: It was removed from the [Appraisals file](https://github.com/mikel/mail/commit/d4ffe72b0b2b0c6c8a49130bfe8e792d0542ac0e) which is what this is auto-generated from. I think it is assumed that 2.99 will always be the [latest 2.x ](https://rubygems.org/gems/mime-types/versions/2.99)

**Comment by jeremy** [comment]
[Backported to 2-6-stable](https://github.com/mikel/mail/commit/22a1528d95557197f58f8ea3088d922d360bdcad)

**Pull request merged**